tests/test_tags.py

```diff
@@ -806,13 +955,23 @@ def test_windows_cpython(self, mock_interpreter_name, monkeypatch):
         abis = tags._cpython_abis(sys.version_info[:2])
         platforms = tags._generic_platforms()
         result = list(tags.sys_tags())
-        interpreter = "cp{major}{minor}".format(
-            major=sys.version_info[0], minor=sys.version_info[1]
+        if sys.version_info[0] >= 10 or sys.version_info[1] >= 10:
```

Comment:
For the cases where it's worth keeping the logic it should probably be broken out into its own test function. Although honestly unless the test is specifically for formatting and thus has hard-coded expectations I say just call `_version_nodot()` directly.